src/lxc/cgroups/cgfsng.c

```diff
@@ -2082,6 +2081,8 @@ static int cgroup_attach_leaf(int unified_fd, int64_t pid)
 		return log_error_errno(-1, errno, "Failed to attach to unified cgroup");
 
 	do {
+		bool rm = false;
+		char attach_cgroup[STRLITERALLEN("lxc-1000/cgroup.procs") + 1];
```

Comment:
Hmm, what's that?

Reply:
As in, what happens if my container name is significantly longer than `lxc-1000`?

Reply:
That's irrelevant. That's the maximum name of the sub-cgroup we create to attach to the container in the unified hierarchy and it is created via `mkdirat()` and has nothing to do with the cgroup that container itself uses.